traveltime-spec.yml

```diff
@@ -55,6 +55,11 @@ components:
       minimum: 60
       maximum: 14400
 
+    RequestTravelTimeForFast:
```

Comment:
The "For" preposition isn't used in other names. This should be named `RequestTravelTimeFast` to match e.g. `RequestTransportationFast` and `ResponseFaresFast`